.github/workflows/chromatic.yml

```diff
@@ -18,12 +18,15 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0 # 👈 Required to retrieve git history
+      - uses: pnpm/action-setup@v3
+        with:
+          version: latest
       - uses: actions/setup-node@v4
         with:
           node-version-file: '.nvmrc'
-          cache: 'yarn'
+          cache: 'pnpm'
       - name: Install dependencies
-        run: yarn
+        run: pnpm install --frozen-lockfile
```

Comment:
Theoretically, this is unnecessary (https://pnpm.io/cli/install#--frozen-lockfile):

> This setting is true by default in [CI environments](https://github.com/watson/ci-info#supported-ci-tools).

But I don't see any harm in leaving it in!

Reply:
Ah didn't realise this! But it's probably fine, and better to be explicit too.